coffeecard/CoffeeCard.Library/Services/v2/AccountService.cs

```diff
@@ -155,6 +159,25 @@ public Task<bool> EmailExistsAsync(string email)
             return _context.Users.AnyAsync(x => x.Email == email);
         }
 
+        public async Task ResendAccountVerificationEmail(ResendAccountVerificationEmailRequest request)
+        {
+            var user = await _context.Users
+                .Where(u => u.Email.Equals(request.Email, StringComparison.OrdinalIgnoreCase))
```

Comment:
Any reason you are putting the lambda in the where, instead of directly in the firstOrDefault?

Reply:
Honestly not. In fact they are equal performance wish, I just find it more explicit but I guess it's just a matter of style